StaticAnalyzer/views/windows.py

```diff
@@ -199,53 +222,98 @@ def _binary_analysis(tools_dir, app_dir):
     bin_path = os.path.join(app_dir, bin_an_dic['bin'])
 
     # Execute strings command
-    # Works only in Mac and Linux by default.
-    args = ["strings", bin_path]
-    bin_an_dic['strings'] = escape(subprocess.check_output(args))
-    bin_an_dic['strings'] = bin_an_dic['strings'].replace("\n", "</br>")
+    bin_an_dic['strings'] = ""
+    str_list = list(strings(bin_path))
+    str_list = set(str_list)  # Make unique # pylint: disable-msg=R0204
+    str_list = [escape(s) for s in str_list]
+    bin_an_dic['strings'] = "</br>".join(str_list)
 
     # Execute binskim analysis if vm is available
-    if settings.WINDOWS_VM_IP is not None:
-        print "[INFO] Windows VM configured."
-        name = _upload_sample(bin_path)
-        bin_an_dic = __binskim(name, bin_an_dic)
-        bin_an_dic = __binscope(name, bin_an_dic)
+    if settings.CURRENT_PLATFROM != 'Windows':
+        if settings.WINDOWS_VM_IP is not None:
+            print "[INFO] Windows VM configured."
+            name = _upload_sample(bin_path)
+            bin_an_dic = __binskim(name, bin_an_dic)
+            bin_an_dic = __binscope(name, bin_an_dic)
+        else:
+            print "[INFO] Windowsc VM not configured in settings.py. Skipping Binskim and Binscope."
+            warning = {
+                "rule_id": "VM",
+                "status": "Info",
+                "desc": "VM is not configured. Please read the readme.md in MobSF/install/windows."
+            }
+            bin_an_dic['results'].append(warning)
     else:
-        print "[INFO] Windowsc VM not configured in settings.py. Skipping Binskim and Binscope."
-        warning = {
-            "rule_id": "VM",
-            "status": "Info",
-            "desc": "VM is not configured. Please read the readme.md in MobSF/install/windows."
-        }
-        bin_an_dic['results'].append(warning)
+        # TODO(Anpassen für windows lokal)
+        print "[INFO] Running lokal analysis."
+
+        global config
+        config = configparser.ConfigParser()
+        config.read('C:\\MobSF\\Config\\config.txt')
```

Comment:
> Or merged with settings.py (Highly recommend as this will be easier for User)
Let me know if I am missing something. 

Sadly I really see a drawback - `setup.py` modifies `config.txt` to save paths of local progs like binskim (so we don't have to modify the path every time binskim jumps to a new version). As far as I understand, changes to `settings.py` made from MobSF are not persistent. (Unless I modify settings.py via loading and adding lines, but I don't think that this will be much better ^^)

> It's ok to place the config anywhere in the Windows VM.
But when it comes to MobSF running to Host OS. The config should be placed under
MobSF/MobSF/config.txt

So for now, I would implement this, since there are less changes required to the existing code.

> but at a time the user will either on VM or host Windows right?

yep, I meant the effort to keep 2 configs up-to-date for us (maintainers). A user will of course always use either the local one or the remote one.